### DISCUSSION TITLE
Adjustable system prompt

### DISCUSSION THREAD

**Discussion by MrCoala** [icereed/paperless-gpt#12] [Ideas]
Hey, 

it would be prety nice if we could adjust the system prompt via the web interface to able to fine tune the results.

**Comment by icereed** [comment]
Thank you for this cool idea. I'm wondering, though, if others would prefer custom prompts via backend configuration. Happy to hear feedback on that.

**Comment by vibe-code-repo** [comment]
Additional customization options are always good :)

**Reply by icereed** [reply]
💯 😁. But the question remains: how? Via env variable (or file) or web UI options?

**Reply by MrCoala** [reply]
Why not via webui?

**Reply by icereed** [reply]
Sure, let's do it. I need to see when I got time for this. Should not be complicated.

**Reply by icereed** [reply]
The first version (without UI) is part of this PR: #15.
I will need to follow up with a UI to modify the prompts and maybe to preview renderings with sample values.

**Reply by icereed** [reply]
The feature is released: https://github.com/icereed/paperless-gpt/discussions/16
